sklearn/feature_extraction/image.py

```diff
@@ -139,6 +138,14 @@ def _to_graph(
     return return_as(graph)
 
 
+@validate_params(
+    {
+        "img": ["array-like"],
+        "mask": [None, np.ndarray],
+        "return_as": [type(np.ndarray), type(sparse.spmatrix)],
```

Comment:
`type(np.ndarray)` and `type(sparse.spmatrix)` are the same, it's just `type`. We could alternatively use the `Options` constraint but we'd have to list all scipy sparse matrix classes which is not ideal. We discussed that with @glemaitre irl and decided to go with just `type`.
```suggestion
        "return_as": [type],
```